moor_generator/lib/src/model/table.dart

```diff
@@ -105,7 +105,7 @@ class MoorTable implements MoorEntityWithResultSet {
 
   /// When non-null, the generated table class will override the
   /// `customConstraints` getter in the table class with this value.
-  final List<String> overrideTableConstraints;
+  List<String> overrideTableConstraints;
```

Comment:
Why did this need to become mutable?

Reply:
I didn't want to break the original functionality (where this field can be null if empty). But, foreign key function use this field after it is initialized (to null). Is it better to have a final growable list? That's mutable too :D